cmd/minikube/main.go

```diff
@@ -121,6 +124,39 @@ func (lb machineLogBridge) Write(b []byte) (n int, err error) {
 	return len(b), nil
 }
 
+// check if a file's size is greater than or equal to max size in KB
+func checkLogFileMaxSize(file string, maxSizeKB int64) bool {
```

Comment:
Comments for funcs should start with the name of the func itself.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
done